tests/test_scripts.py

```diff
@@ -15,6 +15,7 @@
 from cStringIO import StringIO
 import traceback
 from nose.plugins.attrib import attr
+from nose.tools import assert_raises
```

Comment:
Indeed not.

On Tue, Jun 9, 2015, 11:42 Michael R. Crusoe <notifications@github.com>
wrote:

> In tests/test_scripts.py
> <https://github.com/dib-lab/khmer/pull/1057#discussion_r32027639>:
>
> > @@ -15,6 +15,7 @@
> >  from cStringIO import StringIO
> >  import traceback
> >  from nose.plugins.attrib import attr
> > +from nose.tools import assert_raises
>
> Is this used anymore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1057/files#r32027639>.
>